http/src/routing.rs

```diff
@@ -325,6 +333,11 @@ pub enum Route {
         /// The ID of the guild.
         guild_id: u64,
     },
+    /// Route information to create a guild template.
+    CreateTemplate {
+        /// The ID of the guild.
```

Comment:
the

Reply:
i copied the current routing doc for consistency and was planning another PR to fix all that